src/maps/OverlayedMap.cpp

```diff
@@ -279,18 +296,18 @@ void OverlayedMap::drawAirport(const xdata::Airport& airport, double scale) {
             getRunwaysCentre(airport, stitcher->getZoomLevel(), xCentre, yCentre);
             int maxDistance = getMaxRunwayDistanceFromCentre(airport, stitcher->getZoomLevel(), xCentre, yCentre);
             if (maxDistance > ICAO_RADIUS) {
-                drawAirportICAOGeographicRunways(airport);
+                drawAirportICAOGeographicRunways(airport, color);
             } else {
                 drawAirportICAOCircleAndRwyPattern(airport, px, py, ICAO_RADIUS, color);
             }
         }
     }
 
-    drawAirportText(airport, px, py, color);
+    drawAirportText(airport, px, py, mapWidthNM, color - 0x00101010);

```

Comment:
What does that subtraction do? Calculates a good contrast color for a given color? Please add a constant for that as well, another benefit is that the name will make the purpose obvious.